python/metatensor-operations/metatensor/operations/block_from_array.py

```diff
@@ -50,11 +50,20 @@ def block_from_array(array) -> TensorBlock:
         )
 
     samples = Labels.range("sample", shape[0])
+    Labels(
+        names="sample",
+        values=_dispatch.int_array_like(range(shape[0]), array),
+    )
```

Comment:
```suggestion
    samples = Labels(
        names="sample",
        values=_dispatch.int_array_like(range(shape[0]), array),
    )
```

Reply:
Ahh well spotted :)
Eventually it made everything fail so I fixed it